src/code42cli/cmds/profile.py

```diff
@@ -192,7 +193,13 @@ def _prompt_for_allow_password_set(profile_name):
 def _set_pw(profile_name, password):
     c42profile = cliprofile.get_profile(profile_name)
     try:
-        validate_connection(c42profile.authority_url, c42profile.username, password)
+        validate_connection(
+            c42profile.authority_url, c42profile.username, password, None
```

Comment:
Rather than having to explicitly pass in `None` here, we could just make `validate_connection` default to `None` for `totp` arg,